_posts/2022-10-17-10.0.0-release.md

```diff
@@ -0,0 +1,297 @@
+---
+layout: post
+title: "Apache Arrow 10.0.0 Release"
+date: "2022-10-17 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 10.0.0 release. This covers
+over 3 months of development work and includes [**473 resolved issues**][1]
+from [**100 distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 9.0.0 release, Yanghong Zhong, Remzi Yang and Dan Harris
```

Comment:
@kou should I update this with the new PMCs, committers announced or should they be added to the 11.0.0 blog post page? I am not entirely sure around the timelines.

Reply:
Ah, could you add the following people who joined before 10.0.0 release (2022-10-26) https://lists.apache.org/thread/mmfkt6s3b9q9x8kcqy2dcwqbrqd7fcyv ?

- Bogumił Kamiński https://lists.apache.org/thread/p5wfygy4wgvxdntno1xtbmppc8x01onp
- Nicola Crane https://lists.apache.org/thread/j0ymbm4ylfd8h1z5l1n6sjo5vhr5to9w
- Jacob Quinn https://lists.apache.org/thread/3fyckbljtn38nc54cm75d8owoxkm1p3r

We should list the following people in the 11.0.0 blog post because they joined after 10.0.0 release: 

- Ben Baumgold https://lists.apache.org/thread/o97hnm725kbvl27x7fxn4gmzf8swr7vd

Reply:
Thanks updated on this commit: https://github.com/apache/arrow-site/pull/242/commits/8fa16c0a11293ecd9d34400e9415fab67c16a973